README.md

```diff
@@ -1,22 +1,44 @@
 # bedrock-web-session
 Session management for Bedrock Web Apps
 
+# About
+
+A `S
```

Comment:
?

Reply:
Ha! I blame the children. :)

I've updated this now.